code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -626,6 +626,7 @@ GLOBAL_LIST_EMPTY(activated_medevac_stretchers)
 		to_chat(user, "<span class='warning'>[src]'s interface is locked! Only a Squad Leader, Corpsman, or Medical Officer can unlock it now.</span>")
 		return
 	user.drop_held_item()
+	flags_item += NO_VACUUM
 	anchored = TRUE
```

Comment:
This isnt how flags work

Reply:
Of course it's how flags work, otherwise it wouldn't have worked.

Reply:
Dont lecture me if you dont fix it ill just close this

Reply:
Then explain it to me.